LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 Stack Builders Inc.
+Copyright (c) 2017-2019 Stack Builders Inc.
```

Comment:
You could avoid this change every year if you replace it with `2017-Present`
\cc @speedlight